vcpkg/users/port-overlays.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Port overlays
+description: Port overlays
+author: data-queue
+ms.author: danshaw2
+ms.date: 9/20/2023
+ms.prod: vcpkg
+ms.topic: conceptual
+---
+
+# Port Overlays
+
+## Overview
+The default catalog in vcpkg comprises libraries maintained by both the vcpkg team and users. The catalog does not contain the entire universe of libraries, every possible version or patch.
+
+To facilitate the usage of these missing libraries without having to either add them to the default catalog or manage a custom registry, vcpkg uses port overlays. Port overlays are effective when you don't want to worry about versioning and simply want to use a library.
+
+## Overlay Ports
+An overlay port typically refers to a new port or a drop-in replacement for an existing port. In either case, the overlay port takes priority in vcpkg dependency resolution, ensuring it gets selected. However, a potential pitfall is the naming conflict that can arise when multiple overlays introduce the same port.
+
+For instance, `overlay-dir-A` and `overlay-dir-B` may both introduce the same overlay port `zlib`. In this situation, vcpkg resolves `zlib` based on the lookup order in the list of overlay port paths.
+
+## Configuration Syntax
+
+A directory refers to a set of overlay ports and can be specified as follows: 
+- Single overlay port: `<directory>/sqlite3` refers to a single port
+- Directory of overlay ports: `<directory>` refers to a directory of ports
+A valid port must contain both `vcpkg.json` and `portfile.cmake`.
```

Comment:
In general, this paragraph needs to be rewritten to look closer to our current documentation style. Also, since this section is called `Configuration Syntax` it should limit to explain the syntax and not delve on the specifics of what the "overlay ports" directories should contain.

Reply:
I noticed we talked about overlay ports priority resolution in the Registries concepts documentation. I wonder if we should duplicate that here. Thoughts?